client/lib/paths/login/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import config, { isEnabled } from 'config';
+
+export function login( { redirectTo } ) {
+	const url = isEnabled( 'wp-login' ) ? '/login' : config( 'login_url' );
+
+	return redirectTo
+		? addQueryArgs( { redirect_to: redirectTo }, url )
```

Comment:
Will `addQueryArgs` take care of encoding `redirectTo` (something that was performed with `encodeURIComponent` before)?